LocalExtensions.php

```diff
@@ -313,6 +313,10 @@
 	$wgManageWikiSettings['wgDefaultSkin']['options']['Gamepress'] = 'gamepress';
 }
 
+if ( $wmgUseGenealogy ) {
```

Comment:
Need's to be set in LS.php too (default to false) and also needs adding to ManageWiki.php